packages/design-system/src/pages/blocks/HeaderBar.js

```diff
@@ -65,7 +65,7 @@ export default function HeaderBarBlock() {
 						<Dropdown
 							aria-label="Apps switcher"
 							items={[
-								{ onClick: () => {}, label: 'About', type: 'button' },
+								{ onClick: () => setAboutOpen(true), label: 'About', type: 'button' },
```

Comment:
Well that's not great 🤔 We may have to open `Dropdown`'s `items` to ReactElements so you can use `<Modal disclosure=....` here. Something to keep in mind.

Reply:
Yes that's what I also thought, it would be far more convenient. Out of the scope of this IMO though.